src/Microsoft.Management.Infrastructure.CimCmdlets/CimAsyncOperation.cs

```diff
@@ -15,12 +15,12 @@
 namespace Microsoft.Management.Infrastructure.CimCmdlets
 {
     /// <summary>
-    /// <para>
+    /// <param>
```

Comment:
This is not correct.  `<para>` allows defining paragraphs while `<param>` is specifically for parameters to methods.

Reply:
:-) I did not know.

Reply:
I didn't either until I looked it up :)

Personally, would have been better to use <p> instead of <para> for paragraph and just copy HTML.